Remotion/Development/UnitTesting.Compilation/Development.UnitTesting.Compilation.csproj

```diff
@@ -16,5 +16,8 @@
   <ItemGroup>
     <PackageReference Include="System.CodeDom" Version="7.0.0" />
```

Comment:
Is that dependency still needed?

Reply:
Yes because the Rosyln stuff extends the code DOM provider infrastructure